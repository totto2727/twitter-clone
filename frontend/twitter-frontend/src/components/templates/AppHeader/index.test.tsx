import renderer from 'react-test-renderer'
import Header from './index'

test('header template', () => {
  const component = renderer.create(<Header />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
