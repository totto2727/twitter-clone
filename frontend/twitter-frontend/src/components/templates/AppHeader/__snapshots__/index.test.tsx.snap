// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header template 1`] = `
<header>
  header
</header>
`;
